libarchive/adapters/archive_read_add_passphrase.py

```diff
@@ -0,0 +1,5 @@
+import libarchive.calls.archive_read_add_passphrase
```

Comment:
This file is not necessary. It's already integrated in adapters/archive_read.py (where higher-level read operations are integrated).

Reply:
I've just mimicked your original code for other API call. Let me know how to handle it.